src/FSharpLint.Core/Application/Lint.fs

```diff
@@ -454,7 +454,7 @@ module Lint =
             FailedToLoadFile projectFilePath
             |> LintResult.Failure
 
-    /// Lints an entire F# solution by linting all projects specified in the `.sln` file.
+    /// Lints an entire F# solution by linting all projects specified in the `.sln`, 'slnx' or .slnf' file.
```

Comment:
@Numpsy missing one quote; however why not use backticks like with sln?

Reply:
```suggestion
    /// Lints an entire F# solution by linting all projects specified in the '.sln', 'slnx', or '.slnf' file.
```

Reply:
> @Numpsy missing one quote; however why not use backticks like with sln?

Sorry, did it with single quotes at first then changed to backticks to fix the highlighting in the generated docs, but only changed the one in the fsi file that the docs use